docs/ADR-5-cardano-testnet-node-configuration-file.md

```diff
@@ -0,0 +1,39 @@
+# Status
+
+📜 Proposed 2024-09-10
+
+# Context
+
+There is no canonical way to create a node configuration file. `cardano-cli` offers this possibility as part of the bigger [create-cardano](https://github.com/IntersectMBO/cardano-cli/blob/551d9b9f2f244e0d681bf03eaa6d985565ac3a5b/cardano-cli/test/cardano-cli-golden/files/golden/help/latest_genesis_create-cardano.cli#L49) command, but this is ad-hoc.
```

Comment:
Can you highlight more status quo here?

Reply:
In the end I didn't, because the scope changed: we only check the configuration file, we don't create it.